Scripts/script-CBLiveGetFile.yml

```diff
@@ -147,4 +147,5 @@ dependson:
   - cb-command-create
   - cb-command-info
   - cb-file-get
-timeout: 0s
+timeout: 300ns
```

Comment:
ns=nanoseconds ?

Reply:
not sure... was wondering too. but this is the value when you change the automation timeout and download the yml.... suppose to in seconds